src/schema.js

```diff
@@ -1,6 +1,11 @@
 /* eslint-disable class-methods-use-this, arrow-body-style */
 
 import _ from 'lodash';
+import { parseReference } from './json-schema';
+
+export function idForDataStructure(reference) {
+  return `definitions/${parseReference(reference)}`;
```

Comment:
One note, I am using the ID "definitions/{key}" because we are already populating other element keys with unique IDs which could clash. If a user had a swagger document has operationId it could clash with the definition key thus causing a collision. Since Swagger doesn't require that the names are unique across sections.